.github/workflows/fossa.yml

```diff
@@ -0,0 +1,26 @@
+name: FOSSA
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+steps:
```

Comment:
Oh this entire block should be indented once. Prolly why it's not triggering correctly